docker/bootstrap.sh

```diff
@@ -25,6 +25,14 @@ for DB in template_paradedb "$POSTGRES_DB"; do
     CREATE EXTENSION IF NOT EXISTS pg_ivm;
     CREATE EXTENSION IF NOT EXISTS vector;
     CREATE EXTENSION IF NOT EXISTS vectorscale;
+
+    CREATE EXTENSION IF NOT EXISTS postgis;
+		CREATE EXTENSION IF NOT EXISTS postgis_topology;
+		-- Reconnect to update pg_setting.resetval
+		-- See https://github.com/postgis/docker-postgis/issues/288
+		\c
+		CREATE EXTENSION IF NOT EXISTS fuzzystrmatch;
```

Comment:
Could you fix the indentation here? Can you also explain why fuzzystrmatch is needed?

Reply:
Fuzzystrmatch is required for tiger-geocoder as per postgis docs: https://postgis.net/docs/postgis_installation.html#:~:text=For%20tiger%20geocoder%2C%20make%20sure%20you%20have%20postgis%20and%20fuzzystrmatch%20extensions%20available%20in%20your%20PostgreSQL%20instance

I feel like it makes sense to provide the same extensions as the docker-postgis image so I took this SQL snippet from their initdb script.

Reply:
Makes sense, great idea